src/routes/admin/ciraconfig/editCiraConfig.ts

```diff
@@ -57,12 +57,24 @@ export async function editCiraConfig (req, res): Promise<void> {
   }
 }
 
+const handleGenerateRandomPassword = (config: CIRAConfig, newConfig: any, oldConfig: CIRAConfig): CIRAConfig => {
```

Comment:
isn't the newConfig a `CIRAConfig` as well? why any?

Reply:
done